src/test/bridges/notional/NotionalUnit.t.sol

```diff
@@ -0,0 +1,318 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Aztec.
+pragma solidity >=0.8.4;
+
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {BridgeTestBase} from "./../../aztec/base/BridgeTestBase.sol";
+import {NotionalBridgeContract} from "../../../bridges/notional/NotionalBridge.sol";
+import {IWrappedfCashFactory} from "../../../interfaces/notional/IWrappedfCashFactory.sol";
+import {NotionalViews} from "../../../interfaces/notional/INotionalViews.sol";
+import {AztecTypes} from "../../../aztec/libraries/AztecTypes.sol";
+struct Info {
+    uint16 currencyId;
+    uint40 maturity;
+    bool underlying;
+    uint256 interactionNonce;
+}
+
+contract NotionalTest is BridgeTestBase {
+    address public constant ETH = address(0);
+    address public constant CETH = address(0x4Ddc2D193948926D02f9B1fE9e1daa0718270ED5);
+    address public constant DAI = address(0x6B175474E89094C44Da98b954EedeAC495271d0F);
+    address public constant CDAI = address(0x5d3a536E4D6DbD6114cc1Ead35777bAB948E3643);
+    address public constant WETH = address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
+    address public constant USDC = address(0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48);
+    address public constant CUSDC = address(0x39AA39c021dfbaE8faC545936693aC917d5E7563);
+    address public constant WBTC = address(0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599);
+    address public constant CWBTC = address(0xccF4429DB6322D5C611ee964527D42E5d685DD6a);
```

Comment:
You don't need to cast address to address + it feels redundant that you have the addresses here and than you copy them to `tokens` array.